samples/openthread/coap_client/sample.yaml

```diff
@@ -4,23 +4,25 @@ sample:
 tests:
   samples.openthread.coap_client:
     build_only: true
-    platform_allow: nrf5340dk_nrf5340_cpuapp nrf52840dk_nrf52840 nrf52833dk_nrf52833 nrf21540dk_nrf52840
+    platform_allow: nrf52840dk_nrf52840 nrf52833dk_nrf52833 nrf5340dk_nrf5340_cpuapp nrf5340dk_nrf5340_cpuapp_ns
```

Comment:
why did you remove the nrf21540dk_nrf52840 (also in other yaml files)

Reply:
Right, the mergetool got it wrong after a rebase.